maven-model-builder/src/main/java/org/apache/maven/model/building/DefaultModelBuilder.java

```diff
@@ -625,6 +626,15 @@ public DefaultModelBuilder setProfileActivationFilePathInterpolator(
                 versionProcessor);
     }
 
+    /**
+     * @deprecated since Maven 4
+     * @see DefaultModelBuilderFactory#setReportingConverter(ReportingConverter)
```

Comment:
There is nothing to be seen there.

Reply:
All the `setXxx` methods have the same javadoc.  The deprecation is not something new in this case.